{{cookiecutter.project_folder}}/tests/__init__.py

```diff
@@ -0,0 +1,8 @@
+"""Version test for {{ cookiecutter.namespace + '-' + cookiecutter.module_name if cookiecutter.namespace else cookiecutter.module_name }}."""
+
+
+
+def test_version() -> None:
```

Comment:
Should we create the default Python source folder here too and put in an `__init__.py` that this test can run against?

Reply:
Like in this project:

https://github.com/adi-central-ai/cai-ai4ee/blob/dev/backend/tests/__init__.py
https://github.com/adi-central-ai/cai-ai4ee/blob/dev/backend/cai/ai4ee/__init__.py

Reply:
We can, and that was the idea behind adding a test/ directory. I can add the versiontests later if that is ok? Just want to get the repo set up